client/containers/app.jsx

```diff
@@ -1,50 +1,14 @@
 import React, {Component} from 'react';
```

Comment:
this is good work but we have a lot of changes on the appfile that need to wait for this pull pending their implementation